gooey-gui/app/components/bulkProgress/BulkProgressCard.css

```
@@ -0,0 +1,567 @@
+
+.bulk-progress-card-stopped .bulk-progress-dot {
+  background: #9d7b1f;
+}
+
+.bulk-progress-card-error .bulk-progress-dot {
+  background: #b42318;
+}
+
+.bulk-progress-headline {
+  align-self: stretch;
+  font-family: avenir-lt-w01_85-heavy1475544, avenir-lt-w05_85-heavy,
```

Stylelint (17.12.0)

[error] Expected quotes around "avenir-lt-w01_85-heavy1475544" (font-family-name-quotes)

(font-family-name-quotes)